lib/console.js

```diff
@@ -214,6 +219,15 @@ Console.prototype.countReset = function countReset(label = 'default') {
   counts.delete(`${label}`);
 };
 
+Console.prototype.group = function group() {
+  groupIndent++;
```

Comment:
An argument passed to `console.group()` gets used as a label. It's not obvious to me what the best way to distinguish labels from the rest of the output is. However, omitting it entirely is probably not the way to go, so I'll add something like this.